ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/facade/DefaultAuditor.java

```diff
@@ -79,6 +79,9 @@ private boolean shouldAudit(AuditEntry logEntry)
         {
             return !filter.isWhitelisted(logEntry);
         }
+        catch(Exception e) {
+        	return true;
```

Comment:
This solves the immediat issue during start-up. But it will also hide faults in the system long after start-up (potentially causing audits to flod your logs) without giving any trace of the error for the operator.

Would it make sense to narrow it down a bit (UnavailableException?) and perhaps do this in the RoleAuditFilter where the issue originating from? Should we log an error for troubleshooting (and remove call to addSuppressed() until the logback issue is resolved)?

Would it make sense to explore the option of always waiting for schema agreement during start-up. In other words should we remove this condition:
https://github.com/Ericsson/ecaudit/blob/release/c2.2/ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/auth/WhitelistDataAccess.java#L234

WDYT?